src/Workspaces/Core/Portable/Rename/RenameLocations.cs

```diff
@@ -24,33 +24,56 @@ namespace Microsoft.CodeAnalysis.Rename
     /// </summary>
     internal sealed partial class RenameLocations
     {
+        public readonly ISymbol Symbol;
+        public readonly Solution Solution;
+
         public readonly RenameOptionSet Options;
 
+        // possibly null
```

Comment:
how hard will it be to just do `#nullable enable` to help with things like this?

Reply:
i'll try :)

Reply:
done!